### ISSUE TITLE
Joystick detection not working

### ISSUE THREAD

**Issue by tomcw** [AppleWin/AppleWin#778]
Reported by @qkumba.

Prince of Persia fails to detect no joystick when Config->Input "Joystick 1"=Disabled
POP will return no joystick if `JoyReadPosition()` has a `nPdlPos` of 622 or more.

Total Replay uses a modified version of this POP routine:
https://github.com/a2-4am/4cade/blob/master/src/hw.joystick.a
At $25A5 in TR v3.0, similarly returns no joystick found for a value of 622.

NB. Use of higher value (eg. 1000), and do similar for Joystick 2.

**Comment by tomcw** [comment]
EG. a quick patch to `JoyReadPosition()` is
```
	if (joyinfo[joytype[nJoyNum]] == DEVICE_NONE)
		nPdlPos = 1000;
```

**Comment by tomcw** [comment]
In fact it's probably infinite, so just make `nPdlCntrActive` true for the DEVICE_NONE case in `JoyReadPosition()` ...check on real h/w.

**Comment by tomcw** [comment]
Real h/w:
- Power on
- C064.C067- 20 20 20 20
- C070
- C064.C067- A0 A0 A0 A0
- minutes of delay
- C064.C067- A0 A0 A0 A0
- Reset
- C064.C067- A0 A0 A0 A0

**Comment by tomcw** [comment]
@peterferrie - this is now fixed in AppleWin 1.29.12.0 [here](https://github.com/AppleWin/AppleWin/releases/tag/v1.29.12.0).
